src/Graph.hpp

```diff
@@ -38,6 +38,8 @@ class Graph {
 
     static void loadFromEdgeListFile(string fin, string graphName, Graph& g, bool nodesHaveTypes = false);
 	
+	static void connected(uint node1, uint node2, vector<vector<ushort>> edgeList, unsigned int i, Graph& g);
```

Comment:
Few suggestions:
- Rename the function to addEdge or something similar, I believe this is supposed to add edges right?
- Better not use static here, so you would have to pass g as argument.
- Why is edgeList passed by copy to the function? I don't think its needed.

Overall I think the signature should become something like this:
`
void addEdge(uint node1, uint node2, unsiged int weight)
`
And when non-weighted would be:
`
void addEdge(uint node1, uint node2)
`

Reply:
And when you would call it, you would just say ` g -> addEdge(node1, node2)` or use `this->addEdge(node1,node2)`.